toml-rb.gemspec

```diff
@@ -1,9 +1,9 @@
 Gem::Specification.new do |s|
   s.name        = 'toml-rb'
-  s.version     = '0.3.15'
+  s.version     = '0.4.0'
```

Comment:
Since this is a breaking change and it is pretty stable I was thinking to release the version `1.0`

However I am used to create a special commit to bump the version. Please discard the changes on this file, I will handle the version on a different commit.